.github/CODEOWNERS

```diff
@@ -80,6 +80,9 @@ ui/components/component-library      @MetaMask/design-system-engineers
 **/pages/notification*/**                           @MetaMask/notifications
 **/utils/notification.util.ts                       @MetaMask/notifications
 
+# Swaps team to own code for the swaps folder
+ui/pages/swaps                      @MetaMask/swaps-engineers
```

Comment:
should we also add the swaps controller here?

Reply:
Done, and moved to its own folder `app/scripts/controllers/swaps`